### ISSUE TITLE
Support for 'signed-off' commits

### ISSUE THREAD

**Issue by dam0vm3nt** [kevmoo/git#3]
When trying to "parse" signed-off a commit log and exception is thrown:

```
Caught error Bad state: No element
dart:core                                                                                            Iterable.single
package:git/src/commit.dart 59                                                                       Commit._parse
```

**Comment by Stargator** [comment]
@kevmoo If this is fixed, this can be closed, right?

**Comment by kevmoo** [comment]
I have no idea if this is fixed. Does it work for you?

**Comment by Stargator** [comment]
Oh nevermind, I thought b29bef7 was your commit

**Comment by kevmoo** [comment]
It is! looks like it's in!

**Comment by kevmoo** [comment]
Wonder why this didn't get closed then. Weird!

Fixed in https://github.com/kevmoo/git/commit/b29bef7fd9bcb7a50a757ab4c325f44425f6b57c

**Comment by Stargator** [comment]
@kevmoo I'm sorry for the confusion I caused, but I don't think b29bef7 was merged into the repo.

TL/DR: This should remain open and @dam0vm3nt should create a PR with the commit referenced based on the changes to that file. Or I will cause I caused this confusion today.

Longer version:
It was committed to a fork and I looked through all the commits since August 2017 in this repo and nothing seems to merge in that change.

When I type in that hash (as seen at the top of this post), it looks like it's in this repo, but it's not listed as in any branch. But this link https://github.com/dam0vm3nt/git.dart/commit/b29bef7fd9bcb7a50a757ab4c325f44425f6b57c points to where the commit currently resides. In a forked repo.

@kevmoo I'm sorry I inadvertently misled you or confused you.
